src/App.js

```diff
@@ -1,18 +1,45 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import Projects from './Components/Projects';
+import AddProject from './Components/AddProject';
 import './App.css';
 
 class App extends Component {
+  constructor(){
+    super();
+    this.state = {
+      projects: []
+    }
+  }
+
+  componentWillMount(){
+    this.setState({
+      projects: [
+        {
+          title: 'Business Website',
+          category: 'Web Design'
+        },
+        {
+          title: 'Social App',
+          category: 'Mobile Development'
+        },
+        {
+          title: 'Ecommarce shopping Cart',
+          category: 'Web Development'
+        }
+      ]
+    });
```

Comment:
You can achieve this setup in `constructor` as well.

Reply:
In the earlier session, this was in the Constructor only, but he wanted to show lifecycle methods here.
I will update this once after the complete session.